training/benchmarks/mixtral_7b/megatron/README.md

```diff
@@ -0,0 +1,44 @@
+## 模型信息
+- Introduction
+
+The Mixtral-8x7B Large Language Model (LLM) is a pretrained generative Sparse Mixture of Experts. The Mistral-8x7B outperforms Llama 2 70B on most benchmarks we tested.
```

Comment:
remove "The Mixtral-8x7B outperforms xxx" or change it to "Mistral claimed that the MixTral-8x7B outperforms Llama2 xxx"